tests/test_api.rs

```diff
@@ -1023,6 +1024,7 @@ fn set_flags_from_command_line() {
 }
 
 #[test]
+#[ignore] // Terminates the process.
```

Comment:
Should I just remove this test? It exits after printing the help message.

Reply:
IIRC it is a regression test added for a deno bug introduced by a behavioral change in V8's `SetFlagsFromCommandLine` API. Feel free to make a judgement call.

Reply:
Test was removed by another pull request, marking resolved.